pkg/flexvol/flexvol.go

```diff
@@ -348,10 +307,15 @@ func retryGetVolumeNameFromMountPath(k8sPath, dockerPath string) (string, error)
 
 func getMountID(path string) (string, error) {
 	log.Tracef("getMountID called with %v\n", path)
+	// Replace backward slash to forward slash.
```

Comment:
I think the caller has to take care of this(or if caller is windows specific, then no translation is necessary)..

Reply:
getMountID is called at multiple places, it is common for linux and windows. We are simply extracing mountID from the path here and it is handled for both windows and linux . So this looks clean .